packages/launch/src/focused-launch/styles.scss

```diff
@@ -7,34 +7,67 @@
 @import '~@automattic/onboarding/styles/mixins';
 
 .focused-launch__input {
-	.components-base-control__label {
-		font-weight: bold;
+	.components-text-control__input {
+		font-size: 1rem;
+		padding: 10px 16px;
+
+		.components-base-control__label {
+			margin: 0;
+		}
+	}
+}
+
+.focused-launch__mobile-only {
+	@include break-medium {
+		display: none;
+	}
+}
+
+.focused-launch__mobile-commentary {
+	font-size: 0.875rem;
+	color: var( --studio-gray-60 );
+	svg {
+		vertical-align: bottom;
 	}
 }
 
 .focused-launch__label {
-	font-weight: bold;
 	margin-bottom: 8px;
 	display: block;
+	font-size: 1.25rem;
+	color: var( --studio-gray-90 );
+	line-height: 24px;
 }
 
 .focused-launch__section {
-	margin: 20px 0;
+	margin: 0 20px 40px;
+
+	@include break-medium {
+		margin: 0 10px 40px;
+	}
```

Comment:
I would suggest the use of @include onboarding-block-margin; on perhaps the .launch__focused-modal-body element, so you get the same margin treatments on different viewport sizes as seen on gutenboarding and the existing step-by-step launch modal. Not a blocker for this PR.